index.html

```diff
@@ -2595,6 +2595,134 @@ <h3>Data Schemas</h3>
 
       </section>
 
+      <section>
+        <h2>Integrity of Related Resources</h2>
+        <p>
+          When including a link to an external resource in a
+          <a>verifiable credential</a>, it is desirable to know whether
+          the resource that is pointed to is the same at signing time as
+          it is at verification time.  This applies to cases where there
```

Comment:
Doc style is one space after period.
```suggestion
          it is at verification time. This applies to cases where there
```

Reply:
i am old with bad habits ;)

Reply:
I'm with you.  Two spaces is clearly a superior choice!  :-)